### PR TITLE
Clarify external links #147

### LINKED ISSUE #147: Nav Bar Clarify External Links
image

Use the square and arrow icon like this (I found in Cloudflare) to annotate the external links on the nav bar: UBIQUITY INTRO, DOCS, DAO, BLOG

image

Stole the svg from the DOM lol 

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" focusable="false"><path d="M9.996 1.532l.008 1 2.779-.022-4.136 4.136.707.707 4.136-4.136-.023 2.779 1 .008.037-4.508-4.508.036z"></path><path d="M12 12H4V4h4.5V3h-5l-.5.5v9l.5.5h9l.5-.5v-5h-1V12z"></path></svg>
```

### PR DESCRIPTION
**Pull request by Zequez**
Screen Shot 2022-07-15 at 09 22 47

### CODE REVIEW

**Comment by Zequez** [comment]
@pavlovcik Could you add me to the Cloudflare team (if possible) so I can check what's exactly that's not building?

**Comment by 0x4007** [comment]
What's ur email to add

**Comment by 0x4007** [comment]
You were working off an out-of-date commit. I had to merge my fix for Cloudflare Pages 7b90e921e6772dac6f0010f62a85847ae8df2c1f

File: components/layout/Sidebar.tsx [hunk 1]
```diff
@@ -150,6 +150,7 @@ const SocialLinkItem = ({ href, icon, alt }: { href: string; icon: IconsNames; a
 const Item = ({ text, href }: { text: string; href: string; icon: string }) => {
   const router = useRouter();
   const isActive = router.asPath === href;
+  const isExternal = href.startsWith("http");
```
Comment (0x4007) [inline components/layout/Sidebar.tsx:153]: Nice implementation

**Review by 0x4007 — APPROVED** [review]

**Pull request merged**